src/core/adapters/plane.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Plane adapter
+ *
+ * The adapter extracts ticket information from the markup.
+ */
+
+import { match } from "micro-match";
+
+import type { TicketData } from "../types";
+import { $find, $text } from "./dom-helpers";
+import { hasRequiredDetails } from "./utils";
+
+async function scan(url: URL, document: Document): Promise<TicketData[]> {
+  const ogUrl = <HTMLMetaElement> $find("[property~='og:url'][content]", document);
+  if (!ogUrl || ogUrl.content !== "https://app.plane.so/") return [];
+
+  let { issueType } = match("/:workspace/projects/:projectUuid/:issueType/:issueUuid/", url.pathname);
+  if (issueType == "issues") {
```

Comment:
Do other routes and other issue types even exist? If so, are they also plural in the path?

Reply:
Yes but I can not test it as the community edition only has issues and having other issue types is a pro/enterprise feature that costs money...

I'd guess that these custom issue types should also be in plural, but as they be "free form" types

Reply:
Sounds good, thanks for explaining 🙏